src/index.css

```
@@ -63,10 +63,26 @@
   --status-shipped-text: #1e40af;
   --status-done: #d1fae5;
   --status-done-text: #065f46;
+
+  /* Typography stacks */
+  --font-body: 'Inter', 'Noto Sans Thai', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)